android/src/main/java/com/reactnativehmssdk/HmsDecoder.kt

```diff
@@ -337,7 +337,10 @@ object HmsDecoder {
     return roleChangeRequest
   }
 
-  fun getHmsChangeTrackStateRequest(request: HMSChangeTrackStateRequest, id: String): WritableMap {
+  fun getHmsChangeTrackStateRequest(request: HMSChangeTrackStateRequest, id: String): WritableMap? {
+    if (request.mute) {
```

Comment:
@samarthjain10 this logic should be in example app, not within the package
Same for iOS